### PR TITLE
update readme with links to Together web UI

### PR DESCRIPTION
**Pull request by exnx**

### CODE REVIEW

**Comment by athms** [comment]
Thanks, Eric! It looks like we are using different spellings for Together AI (sometimes only Together, then TogetherAI, and even TogetheAI). Can we streamline our wording and just say Together AI?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Evo is a biological foundation model capable of long-context modeling and design
 Evo uses the [StripedHyena architecture](https://github.com/togethercomputer/stripedhyena) to enable modeling of sequences at a single-nucleotide, byte-level resolution with near-linear scaling of compute and memory relative to context length.
 Evo has 7 billion parameters and is trained on OpenGenome, a prokaryotic whole-genome dataset containing ~300 billion tokens.
 
-We describe Evo in the paper [“Sequence modeling and design from molecular to genome scale with Evo”](https://www.biorxiv.org/content/10.1101/2024.02.27.582234v1) and in the [accompanying blog post](https://arcinstitute.org/news/blog/evo).
+We describe Evo in the the paper [“Sequence modeling and design from molecular to genome scale with Evo”](https://www.biorxiv.org/content/10.1101/2024.02.27.582234v1) and in the [accompanying blog post](https://arcinstitute.org/news/blog/evo).
```
Comment (brianhie) [inline README.md:9]: Fix typo, two "the" words

File: README.md [hunk 2]
```diff
@@ -125,7 +126,10 @@ model = AutoModelForCausalLM.from_pretrained(
 
 ## Together API
 
-Evo will also be soon available via an API by [TogetherAI](https://www.together.ai/).
+Evo is available through TogetheAI with a [web UI](https://api.together.xyz/playground/language/togethercomputer/evo-1-131k-base), where you can interaface and generate DNA sequences with a chat-like interface.
```
Comment (brianhie) [inline README.md:129]: Fix typos: "TogetheAI" and "interaface"

**Review by brianhie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (exnx) [reply README.md:9]: thanks!

File: README.md (hunk 2 above)
Comment (brianhie) [reply README.md:129]: Also, we repeat interface twice, maybe just "where you can generate DNA sequences with a chat-like interface."

Comment (exnx) [reply README.md]: ok, updated with new commit :) thanks!

**Pull request merged**